demo/Signs.swift

```diff
@@ -1067,6 +1026,589 @@ extension Sign {
                 return Asset.Signs.warningRoadworksCN
             case .warningSecondRoadLeft:
                 return Asset.Signs.warningSecondRoadLeftCN
+            default: return nil
+            }
+        case .UK, .other:
+            switch type {
+            case .unknown:
+                return nil
+            case .speedLimit:
+                switch number {
+                case 5:
+                    return over ? Asset.Signs.overSpeedLimit5EU : Asset.Signs.speedLimit5EU
```

Comment:
Current convention suppose `speedLimitEU5Over` naming. Please consult with @rokkeds to align with this convention.